packages/web-frontend/src/screens/mobile/RegionScreen/index.js

```diff
@@ -291,7 +291,7 @@ const mapStateToProps = state => {
 };
 
 const mapDispatchToProps = dispatch => ({
-  onChangeMeasure: measureId => dispatch(setMeasure(measureId)),
+  onChangeMapOverlay: mapOverlayId => dispatch(setMapOverlay(mapOverlayId)),
   onClearMeasure: () => dispatch(clearMeasure()),
```

Comment:
@kael89 TOGGLE_MEASURE_EXPAND is used with onToggleMeasureExpand(), will delete other two that you've found.

I left these measure variables for PHX-1 because I feel like they will be handled and changed slightly different from map overlay for multiple map overlay checkbox (but `onClearMeasure` is not this case..). Do you think we can still keep others for now?

Reply:
Fair enough! Let's leave them as they are for now.

Good point about `onToggleMeasureExpand`, I just realised it's used in `RegionScreen` 👍